protocol/contracts/profit/src/state/mod.rs

```diff
@@ -129,27 +107,23 @@ impl ConfigManagement for State {
 }
 
 impl State {
-    pub fn new(config: Config) -> Self {
-        Self(StateEnum::OpenTransferChannel(OpenTransferChannel::new(
-            config,
-        )))
+    pub(crate) fn start(config: Config, dex: ConnectionParams) -> (Self, Batch) {
```

Comment:
The visibility of this fn is limited to the visibility of the `struct` it is defined in. Therefore no need to specialize its visibility. Please remove.